debian/copyright

```diff
@@ -0,0 +1,52 @@
+
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: paflib
+Source: https://github.com/leitao/paflib_deb
+
+Files: *
+Author:  Carlos Eduardo Seo (ceseo) <contact@carlosseo.com>
```

Comment:
Here, it would also be better to use what's in configure.ac, similarly to what I mentioned in the file debian/control. So, it would be better to have this line as:

    Author:  Tulio Magno Quites Machado Filho <tulioqm@br.ibm.com>